app/controllers/ping_things_controller.rb

```diff
@@ -6,5 +6,7 @@ def index
                             .includes(:user)
                             .order(created_at: :desc)
                             .first(240)
+    @ping_things_count = @ping_things.count
```

Comment:
```suggestion
    @ping_things_count = @ping_things.length
```
to avoid not needed (I suppose) query to db